tls/s2n_server_new_session_ticket.c

```diff
@@ -74,3 +75,75 @@ int s2n_server_nst_send(struct s2n_connection *conn)
 
     return 0;
 }
+
+int s2n_tls13_server_nst_send(struct s2n_connection *conn)
+{
+    notnull_check(conn);
+
+    /* Write message type because session resumption in TLS13 is a post-handshake message */
+    GUARD(s2n_stuffer_write_uint8(&conn->handshake.io, TLS_SERVER_NEW_SESSION_TICKET));
+
+    struct s2n_stuffer_reservation message_size = { 0 };
+    GUARD(s2n_stuffer_reserve_uint24(&conn->handshake.io, &message_size));
+
+    /* Ticket lifetime is the minimum between key lifetime, session state lifetime, and one week */
```

Comment:
This calculation is probably worth breaking out into its own method. We'll likely want to reuse it when checking if a received ticket is valid, and it'll make testing easier.